src/Lavary/Menu/Builder.php

```diff
@@ -408,7 +408,9 @@ protected function getRoute($options)
 	{
 		if (is_array($options))
 		{
-			return \URL::route($options[0], array_slice($options, 1));
+			$absolute = isset($options['absolute']) ? $options['absolute'] : true;
```

Comment:
I think this is a good idea, but I really think using both `$options['absolute']` and `$options[0]` on the same array is not a great idea. I'm going to bump this to v1.7.2 and review the alternatives. It may be your solution is the best option, but I'd like to see what else can be done.